### PR TITLE
luci-app-aliyundrive-webdav: tidy up Makefile

### PR DESCRIPTION
**Pull request by Beginner-Go**

### CODE REVIEW

**Comment by messense** [comment]
Thanks for the PR, can you rebase onto main branch? I want to make sure it the OpenWrt packages compiles. See https://github.com/messense/aliyundrive-webdav/commit/976af29806fee33c927338e6fae014196c9fdb1a

**Comment by Beginner-Go** [comment]
Ok ，wait a moment

**Comment by messense** [comment]
Oops, I made a mistake (https://github.com/messense/aliyundrive-webdav/commit/9b13438ee1d8855f3567ceb0b3c7d4b1cfdc5be9), could you please rebase again.

**Comment by Beginner-Go** [comment]
ok , I pushed it again

File: openwrt/luci-app-aliyundrive-webdav/luasrc/view/aliyundrive-webdav/aliyundrive-webdav_log.htm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%+cbi/valueheader%>
```
Comment (messense) [inline openwrt/luci-app-aliyundrive-webdav/luasrc/view/aliyundrive-webdav/aliyundrive-webdav_log.htm:1]: Is there any reason there files are moved around like this?

**Review by messense — COMMENTED** [review]
Thanks! I also wanted to do it like this before but failed. 
Just one question, it builds a `luci-i18n-aliyundrive-webdav-zh-cn_git-21.253.58402-4d5abca_all.ipk`, can we set the package version to the same as the main luci package? For example, I'd like to have it output `luci-i18n-aliyundrive-webdav-zh-cn_0.1.23-0_all.ipk`

**Comment by Beginner-Go** [comment]
> Thanks! I also wanted to do it like this before but failed.
> Just one question, it builds a `luci-i18n-aliyundrive-webdav-zh-cn_git-21.253.58402-4d5abca_all.ipk`, can we set the package version to the same as the main luci package? For example, I'd like to have it output `luci-i18n-aliyundrive-webdav-zh-cn_0.1.23-0_all.ipk`

这是由Makefile文件决定的 我所用的是luci写法，这样语言包和插件包会分离，这也是官方标准的写法

File: openwrt/luci-app-aliyundrive-webdav/luasrc/view/aliyundrive-webdav/aliyundrive-webdav_log.htm (hunk 1 above)
Comment (Beginner-Go) [reply openwrt/luci-app-aliyundrive-webdav/luasrc/view/aliyundrive-webdav/aliyundrive-webdav_log.htm:1]: 没有移动文件，只是改了文件名，这也是官方命名规则

**Comment by Beginner-Go** [comment]
还有一些小问题要解决 暂时不要merged

**Comment by KFERMercer** [comment]
> Thanks! I also wanted to do it like this before but failed.
> Just one question, it builds a `luci-i18n-aliyundrive-webdav-zh-cn_git-21.253.58402-4d5abca_all.ipk`, can we set the package version to the same as the main luci package? For example, I'd like to have it output `luci-i18n-aliyundrive-webdav-zh-cn_0.1.23-0_all.ipk`

参见这个脚本: https://github.com/openwrt/openwrt/blob/master/scripts/getver.sh

**Comment by messense** [comment]
> 参见这个脚本: https://github.com/openwrt/openwrt/blob/master/scripts/getver.sh

应该是这里的 `PKG_PO_VERSION` 来的？https://github.com/openwrt/luci/blob/db75909587a42e2a7bcf9fcdf077bba977c94c32/luci.mk#L285

**Review by messense — APPROVED** [review]

**Pull request merged**